server/src/routes/jiraIssues.js

```diff
@@ -0,0 +1,47 @@
+import express from 'express';
+import JiraApi from 'jira-client';
+import config from '../config';
+
+const jira = new JiraApi({
+  protocol: 'https',
+  host: config.jira.host,
+  username: config.jira.username,
+  password: config.jira.password,
+  apiVersion: '2',
+  strictSSL: true,
+});
+
+const router = express.Router();
+
+router.get('/', (req, res, next) => {
+  jira.searchJira("project = CC AND status in (Open, 'In Progress', Reopened)")
```

Comment:
As the route is something between Controller and View, please move as much as possible data-fetching and business logic to a separate "Service". 
Keep only the response building here.